ArkGameExample/TankGame/TankGameEntityCreator.swift

```diff
@@ -28,7 +28,8 @@ struct TankGameEntityCreator {
     static func createJoyStick(center: CGPoint, tankEntity: Entity, in ecsContext: ArkECSContext,
                                eventContext: ArkEventContext) {
         ecsContext.createEntity(with: [
-            JoystickCanvasComponent(radius: 40, areValuesEqual: { _, _ in true })
+            JoystickCanvasComponent(radius: 40)
+                .shouldRerender { _, _ in true }
```

Comment:
I think this should be changed to `false`

Reply:
u are great